tests/Bot.Builder.Community.Adapters.Tests/TwitterAdapterTests.cs

```diff
@@ -78,17 +79,53 @@ public async Task SendActivitiesAsyncShouldReturnEmptyResponsesWithEmptyActiviti
             }
         }
 
+        [TestMethod]
+        public async Task SendActivitiesAsyncShouldReturnEmptyResponsesWithActivities()
+        {
+            var options = new TwitterOptions
+            {
+                WebhookUri = "uri",
+                AccessSecret = "access-secret",
+                AccessToken = "access-token",
+                ConsumerKey = "consumer-key",
+                ConsumerSecret = "consumer-secret",
+                Environment = "env",
+                Tier = TwitterAccountApi.PremiumFree
+            };
+
+            _testOptions.SetupGet(x => x.Value).Returns(options);
+
+            var adapter = new TwitterAdapter(_testOptions.Object);
+            var activity = new Activity();
+
+            using (var turnContext = new TurnContext(adapter, activity))
+            {
+                var activities = new List<Activity>()
+                {
+                    new Activity()
+                    {
+                        Type =  ActivityTypes.Message,
+                        Text = "Test",
+                        Recipient = new ChannelAccount(),
+                    }
+                };
+
+                var result = await adapter.SendActivitiesAsync(turnContext, activities: activities.ToArray(), default);
+                Assert.IsTrue(result.Length > 0);
```

Comment:
Maybe we can assert the exact number of responses expected and their id.

Reply:
I can change it to be equal to 1, but the response content only return the ID with a null value, I look up to the code and it is because of unauthorized error, that's because I send wrong credentials just for covering the required block code.